.ci/test-spelling-unknown-words.sh

```diff
@@ -8,20 +8,15 @@ set -e
 spellchecker='config/jsoref-spellchecker'
 temp='.ci-temp'
 whitelist_path="$spellchecker/whitelist.words"
-dict="$temp/english.words"
+# english.words is taken from rpm:
+# https://rpmfind.net/linux/fedora/linux/development/rawhide/Everything/aarch64/os/Packages/w/"
+# "words-.*.noarch.rpm"
+dict="$spellchecker/english.words"
 word_splitter="$temp/spelling-unknown-word-splitter.pl"
 run_output="$temp/unknown.words"
 
 mkdir -p $temp
 
-if [ ! -e "$dict" ]; then
-  echo "Retrieve cached english.words from checkstyle.sourceforge.io"
-  # english.words is taken from rpm:
-  # https://rpmfind.net/linux/fedora/linux/development/rawhide/Everything/aarch64/os/Packages/w/"
-  # "words-.*.noarch.rpm"
-  curl --fail-with-body -k https://checkstyle.sourceforge.io/reports/english.words -o $dict
```

Comment:
we have one more download:
```
  w_location='https://raw.githubusercontent.com/jsoref/spelling/master/w'
  curl --fail-with-body -s "$w_location" |\
```

so probably we can just move english.words  to https://raw.githubusercontent.com